lib/screenplay_web/controllers/config_controller.ex

```diff
@@ -89,6 +89,23 @@ defmodule ScreenplayWeb.ConfigController do
     })
   end
 
+  def publish(conn, %{
+        "place_id" => place_id,
+        "app_id" => app_id,
+        "hidden_from_screenplay_ids" => hidden_from_screenplay_ids
+      }) do
+    app_id_atom = String.to_existing_atom(app_id)
+
+    case PermanentConfig.publish_pending_screens(
+           place_id,
+           app_id_atom,
+           hidden_from_screenplay_ids
+         ) do
+      :ok -> send_resp(conn, 200, "OK")
+      _ -> send_resp(conn, 400, "Could not publish screens")
```

Comment:
Should this technically be a 500 error? I thought 4XX errors are typically given when the client has provided a bad request, whereas this case seems to indicate a server-side issue

Reply:
Yeah, you're totally right. Changed that and a couple in `put` as well. I left `:version_mismatch` as a 400 error considering the version we check comes from the request. Seemed like a good use of 400, but let me know if you disagree.

Reply:
Yeah I think `:version_mismatch` does make sense as a 400